5.1.thats_the_way.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def thats_the_way(path):      # from 5.1
+    res = []
+    for file in os.listdir(path):
+        file_path = os.path.join(path, file)
+        if os.path.isfile(file_path):
+            file_name = os.path.basename(file_path)
+            if file_name.startswith("deep"):
+                res.append(file_name)

```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.